_posts/en/newsletters/2020-07-29-newsletter.md

```diff
@@ -125,30 +125,71 @@ release candidates.*
-  relay][topic package relay].  See our [extended summary][news104
-  bip339] of BIP339 for details.
-
-    After being merged into Bitcoin Core's master branch, this PR was
-    [discussed][meeting xscript] during the weekly Bitcoin Core
-    Development Meeting in relation to taproot activation.  It was noted
-    that the version of taproot merged into Bitcoin Core (see PR
-    [#17977][Bitcoin Core #17977]) will likely be based on wtxid relay.
-    This will make it more complicated to backport taproot to the
-    current 0.20.x branch of Bitcoin Core and so possibly ensure that
-    the earliest release of taproot activation logic will be in the
-    Bitcoin Core 0.21.x branch, whose major release is roughly
-    [expected][Bitcoin Core #18947] in December 2020.  A proposed
-    alternative for backporting taproot to the current 0.20.x branch,
-    either so miner signaling can start sooner or just so it's easier
-    for miners to upgrade, would be to only backport the logic for
-    verifying taproot transactions in blocks and not the wtxid
-    transaction relay logic.
+- [Bitcoin Core #18044][] adds support for witness txid (wtxid)
+  transaction inventory announcements (`inv`) and requests (`getdata`)
+  as described in BIP339 (see [Newsletter #104][news104 bip339]).  Prior
+  to this merge, all Bitcoin nodes announced new unconfirmed
+  transactions to their peers by the transaction's txid.  However, txids
+  don't commit to the witness data in segwit transactions, so a node who
+  downloaded an invalid or unwanted segwit transaction can't safely
+  assume that any transaction with that same txid is also invalid or
+  unwanted.  That means nodes may waste bandwidth by downloading the
+  same bad transaction over and over from each peer announcing that
+  transaction.
+
+    So far this hasn't been an issue---honest peers usually don't
+    announce transactions they wouldn't accept themselves, so only a
+    disruptive peer that wanted to waste its own upload bandwidth would
+    advertise invalid or unwanted transactions.  However, one type of
+    unwanted transaction today are spends of v1 segwit UTXOs---the types
+    of spends the [BIP341][] specification of [taproot][topic taproot]
+    plans to use.  If taproot activates, this means newer taproot-aware
+    nodes will advertise taproot spends to older taproot-unaware nodes.
+    Each time one those taproot-unaware nodes receives a
+    taproot-spending transaction, it will download it, realize it uses
+    v1 segwit, and throw it away.  This could be very wasteful of
+    network bandwidth, both for older taproot-unaware nodes and newer
+    taproot-aware nodes.  This same problem applies to other proposed
+    changes to network relay policy.
+
+    The solution implemented in this merged PR is to announce
+    transactions by their wtxid---which includes a commitment to the
+    witness data for segwit transactions.  It is expected that the
```

Comment:
'expected' is maybe a bit strong. When I asked Pieter about whether we'd only relay taproot using wtxid relay, he said that he hadn't really thought about it. Perhaps you could soften this to something like "A taproot implementation in Bitcoin Core could then only relay transactions by their wtxid ..."